backend/priv/repo/migrations/20200401144955_change_likes_for_reactions.exs

```diff
@@ -0,0 +1,20 @@
+defmodule Til.Repo.Migrations.ChangeLikesForReactions do
+  use Ecto.Migration
+
+  def change do
+    drop unique_index(:likes, [:user_id, :post_id], name: :user_post_pair)
+
+    drop table(:likes)
+
+    create table(:reactions) do
+      add :user_uuid, :string
+      add :type, :string
+      add :user_id, references(:users, on_delete: :nothing), null: false
+      add :post_id, references(:posts, on_delete: :nothing), null: false
+
+      timestamps()
+    end
+
+    create unique_index(:reactions, [:user_id, :post_id, :type], name: :user_post_reaction)
```

Comment:
to better utilize index you can swap columns order: `[:post_id, :user_id, :type]`
most likely we will only fetch data based on post_id (posts index/show page), in the current form index wouldn't be used